Sources/Workspace/InitPackage.swift

```diff
@@ -239,7 +318,8 @@ public final class InitPackage {
         // Create a tools version with current version but with patch set to zero.
         // We do this to avoid adding unnecessary constraints to patch versions, if
         // the package really needs it, they should add it manually.
-        let version = InitPackage.newPackageToolsVersion.zeroedPatch
+        let version = packageType == .macro ? ToolsVersion.vNext
```

Comment:
Also something to keep in mind to update later.

Reply:
Right, we'll update this when the `macro` target gets a proper Swift version instead of 999.